com.microsoft.mrtk.spatialmanipulation/Constraints/ConstraintManager.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.MixedReality.Toolkit.SpatialManipulation
     /// <summary>
     /// Manages constraints for a given object and ensures that Scale/Rotation/Translation 
     /// constraints are executed separately.
+    /// We're looking to rework this system in the future. These existing components will be deprecated in then. 
```

Comment:
The last sentence (throughout these components) might have an extra word?

```suggestion
    /// We're looking to rework this system in the future. These existing components will be deprecated then. 
```